app/templates/snippet-management/edit/edit-snippet.hbs

```diff
@@ -84,6 +84,10 @@
     <:aside>
       {{#if this.editor}}
         <Sidebar as |Sidebar|>
+          <ConnectArticlesToImportedResource
+            @controller={{this.editor}}
+            @importedResource={{this.importedDecisionUri}}
```

Comment:
I don't think it works well to pass in a URI, as it ignores any existing imported resources in the document. This also means matching URIs, so if there are e.g. 3 articles without an imported resource link in the snippet, we get 3 separate imported resources with the same URI. The issue of multiple imported resources with the same URI doesn't cause any issues that I can find so far though.

Reply:
Ok so:
- The issue with multiple imported resources with the same URI is now solved.
- I indeed did not take into account already-exisiting imported resources. But I don't know if we already added imported resources to the IV snippets on QA and PROD. Additionally, we want to make sure we use the same imported-resource URI across the whole snippet-list.